packages/discord.js/src/structures/Modal.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const { Modal: BuildersModal } = require('@discordjs/builders');
+const Transformers = require('../util/Transformers');
+
+class Modal extends BuildersModal {
+  constructor(data) {
+    super(Transformers.toSnakeCase(data));
```

Comment:
Won't this collide with the PR that makes it a client options?

Reply:
Djs builders always use the built-in transformer, but regular builders don't transform at all.

Reply:
He means #7476, which was landed 23 hours ago.

But yeah, they're not using in the builder constructors, we can't reference the client from here anyways.